src/dialect/druidDialect.ts

```diff
@@ -243,4 +247,16 @@ export class DruidDialect extends SQLDialect {
   public lookupExpression(base: string, lookup: string): string {
     return `LOOKUP(${base}, ${this.escapeLiteral(lookup)})`;
   }
+
+  public ipMatchExpression(columnName: string, searchString: string): string {
+    return `IP_MATCH(${this.ipParse(columnName)}, ${this.escapeLiteral(searchString)})`;
```

Comment:
Using `IP_PARSE` here because in Pivot need to use `IP_STRINGIFY` on ip/ip prefix dimensions since `group by` clause cannot be applied to `COMPLEX<ipAddress>` (or `COMPLEX<ipPrefix>`).